dcos/config.py

```diff
@@ -213,6 +213,25 @@ def get_config_schema(command):
     return subcommand.config_schema(executable, command)
 
 
+def get_property_description(section, subkey):
+    """
+    :param section: section of config paramater
+    :type section: str
+    :param subkey: property within 'section'
+    :type param: str
+    :returns: description of section.subkey, or title if no description
+    :rtype: str
+    """
+
+    schema = get_config_schema(section)
+    property_info = schema["properties"].get(subkey)
+    if property_info is not None:
+        return property_info.get("description", "title")
```

Comment:
Copy and paste of a previous comment:

> Right but get('key', 'default value') on a dict tries to find the key 'key'. If it can't find it, it returns 'title', no?